### PR TITLE
Document how to look at dask worker logs from GCP

### PR DESCRIPTION
**Pull request by yuvipanda**
Ref https://2i2c.freshdesk.com/a/tickets/426

### CODE REVIEW

File: docs/howto/troubleshoot/logs.md [hunk 1]
```diff
@@ -88,15 +90,26 @@ errors can be frustrating!
...
 labels.k8s-pod/app_kubernetes_io/name="dask-gateway"
 resource.labels.namespace_name="<namespace>"
 ```
 
+To look at just the *scheduler* and *worker* logs of a specific user, you can
+try:
+
+```
+labels.k8s-pod/app_kubernetes_io/name="dask-gateway"
+resource.labels.namespace_name="<namespace>"
```
Comment (jbusecke) [inline docs/howto/troubleshoot/logs.md:106]: I am not entirely sure what `namespace` is in the context of these queries. Adding a concrete example might help somebody like me understand this quicker.

Comment (yuvipanda) [reply docs/howto/troubleshoot/logs.md:106]: good point, @jbusecke! For quick reference in most cases this is 'prod', and the value is the 'name' for the hub assigned here: https://github.com/2i2c-org/infrastructure/blob/a4aefed81700fcbe2a6224b30a61e8abd155ba76/config/clusters/leap/cluster.yaml#L23.

I'll find a way to document this and how users can determine it

Comment (jbusecke) [reply docs/howto/troubleshoot/logs.md:106]: Awesome thanks

**Review by consideRatio — APPROVED** [review]
Nice this LGTM!

**Review by GeorgianaElena — APPROVED** [review]

**Comment by yuvipanda** [comment]
I'm going to merge this as is for now, can add additional docs on finding namespaces later.

**Pull request merged**